client/components/LearnerViewDirective.js

```diff
@@ -21,104 +21,174 @@ tie.directive('learnerView', [function() {
     restrict: 'E',
     scope: {},
     template: `
-      <h3>Exercise: {{title}}</h3>
-
-      <div class="tie-learner-view-left-column">
-        <div class="tie-previous-instructions">
-          <div ng-repeat="previousInstruction in previousInstructions track by $index">
-            <p ng-repeat="paragraph in previousInstruction track by $index">
-              {{paragraph}}
-            </p>
-            <hr>
+      <div id="tie-exercise-ui-outer" class="tie-exercise-ui-outer">
```

Comment:
Er, I might not have been clear previously. Can we drop the IDs throughout, unless they're really needed?